### PR TITLE
feat(docs): add API documentation back

### PR DESCRIPTION
**Pull request by LouisBrunner**
## Description

See it here: https://docs.aica.tech/api/

Readd the API removed in #34

The documentation is built in-step with the Backend API and then pinned in this repository. The `docs/api.version` file points to the version to be deployed (currently using a feature branch due to the lack of release with this feature).

## Review guidelines

Estimated Time of Review: 2 minutes

### CODE REVIEW

**Comment by LouisBrunner** [comment]
> I guess you need to change the branch but otherwise good

For the API version you need? We need a release then

**Comment by domire8** [comment]
We could do that after https://github.com/aica-technology/backend-api/pull/85 no? @eeberhard

**Comment by eeberhard** [comment]
> We could do that after [aica-technology/backend-api#85](https://github.com/aica-technology/backend-api/pull/85) no? @eeberhard

Yes thanks, that's partly why I was holding off on approval. I will get that PR resolved today so that we can release the backend API and then get this in.

File: docs/api.version [hunk 1]
```diff
@@ -0,0 +1 @@
+feature-docs-46
```
Comment (domire8) [inline docs/api.version:1]:
```suggestion
v1.0.0
```

**Pull request merged**